### ISSUE TITLE
Also highlight selected shape's contact info in Testbed

### ISSUE THREAD

**Issue by louis-langholtz** [louis-langholtz/PlayRho#33]
### Expected/Desired Behavior or Experience:
Something like the following where the contact info for a selected shape is highlighted over the non-selected shapes (along with highlighting the selected shape) :
![screen shot 2017-08-06 at 11 06 40 am](https://user-images.githubusercontent.com/9611010/29005396-fc600b74-7a97-11e7-8b0f-bd0da634329d.png)

### Actual Behavior:
Like the above picture but without the selected shape's contact points, normals, and impulses highlighted.

### Steps to Reproduce the Actual Behavior:
Run Testbed demo with Contact Points, Contact Normals, Contact Impulses, and Friction Impulses toggles selected.

### Related Issues
This may help with diagnosing problems like the Tumbler squishiness described in issue #10 .

**Comment by louis-langholtz** [comment]
**Note:**

The contact dots (the blue dots) locations admittedly look weird in the picture associated with this issue. Either they're in the wrong places or they're where the contact point locations were at the start of the time step and Erin's Box2D code doesn't look so misaligned (which is also odd).

**In any case, this issue isn't for solving this misalignment. This issue is simply for highlighting the impulses associated with a selected shape.**

**Comment by louis-langholtz** [comment]
Regarding the previously mentioned oddity of dot location...

Turns out the dot locations appear differently in Erin's 2.3.2 Box2D because contact information in Erin's Box2D code gets updated regardless of whether the world step is advancing any time while contact information is only updated in PlayRho if time's advancing.

I made a quick mod to Erin's code so that it updated the contact data only when the step's time was greater than zero and only reset the Testbed point data under the same condition and sure enough the dots appear in similarly odd looking positions:

![screen shot 2017-08-06 at 11 52 30 am](https://user-images.githubusercontent.com/9611010/29005708-4ae5a38e-7a9e-11e7-8ece-9eb43c3d7d31.png

So what we're seeing is the dots at the location when the `PreSolve` is called (which is before bodies get moved) and the bodies at the location after all the remaining processing for the world step has finished. So things look out of sync and weird to me but now that I understand why, these locations make more sense. A separate issue should be opened if this "sync" related issue should be addressed further.

Closing this issue as completed now.
